eclipse-theia/theia
# Add keybindings with numpad keys for Zoom commands and handle multiple keybindings per command

#### What it does

- Add keybindings with the + and - keys from the numpad for the `Zoom In` and `Zoom Out` commands, in the electron app (Fixes #11239)
- Display all available keybindings for a command in the Keyboard Shortcuts dialog and handle multiple keybindings in the `keymaps.json` (Fixes #11240)

#### How to test


Zoom In and Out should work with the + and - keys from the numpad.

To test the handling of multiple keybindings for one command:
1) Open the Keyboard Shortcuts editor and `keymaps.json`
2) Search for `Zoom In` command in the editor -> two entries should show
3) Modify one of the entries and change the keybinding in the editor -> two new entries should be visible in `keymaps.json`, one for the new keybinding and one that disables the default one
4) Save `keymaps.json` to update the UI
5) Modify the second (default) entry for the `Zoom In` command in the editor -> two new entries should be visible in `keymaps.json`, one for the new keybinding and one that disables the default one
6) Save `keymaps.json` to update the UI
7) Modify the second (default) entry again in the editor -> one entry should update in the `keymaps.json` (the one that was added at step 5)
8) Save `keymaps.json` to update the UI
9) Reset one of the `Zoom In` commands in the editor -> both commands should return to their default values and the command should be removed completely from `keymaps.json`

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
